lighthouse-plugin-publisher-ads/test/smoke/fixtures/layout-shift.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<!--
+* Copyright 2021 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>GPT Layout Shift Test</title>
+    <style>
+      p {
+        font-size: 50px;
+      }
+    </style>
+    <script>googletag = window.googletag || {cmd: []};</script>
+    <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
+    <script>
+      googletag.cmd.push(() => {
+          googletag.defineSlot('/6355419/Travel', [728, 90], 'div0')
+              .setTargeting("test","infinitescroll")
+              .addService(googletag.pubads());
+      });
+      googletag.cmd.push(() => {
+          googletag.defineSlot('/6355419/Travel', [728, 90], 'div1')
+              .setTargeting("test","infinitescroll")
+              .addService(googletag.pubads());
+      });
+      googletag.cmd.push(function() {
+        // googletag.pubads().enableSingleRequest();
+        googletag.enableServices();
+        googletag.pubads().addEventListener('slotRenderEnded', function(evt) {
+          setTimeout(function() {
+            const el = document.getElementById(evt.slot.getSlotElementId());
+            el.style.height = '40px';
```

Comment:
It seems like this would shrink the slot, but the JS is measuring expansions?

Reply:
It shrinks the slot after expansion

Reply:
Ah, I see. Thanks!